packages/dashboard-v2/src/pages/upgrade.js

```diff
@@ -0,0 +1,132 @@
+import * as React from "react";
+import bytes from "pretty-bytes";
+import styled from "styled-components";
+
+import { useUser } from "../contexts/user";
+import { PlansProvider } from "../contexts/plans/PlansProvider";
+import useActivePlan from "../hooks/useActivePlan";
+import DashboardLayout from "../layouts/DashboardLayout";
+import { Panel } from "../components/Panel";
+import Slider from "../components/Slider/Slider";
+import { CheckmarkIcon } from "../components/Icons";
+import { Button } from "../components/Button";
+
+const SLIDER_BREAKPOINTS = [
+  {
+    name: "xl",
+    scrollable: true,
+    visibleSlides: 4,
+  },
+  {
+    name: "lg",
+    scrollable: true,
+    visibleSlides: 3,
+  },
+  {
+    name: "sm",
+    scrollable: true,
+    visibleSlides: 2,
+  },
+  {
+    scrollable: true,
+    visibleSlides: 1,
+  },
+];
+
+const PlanSummaryItem = ({ children }) => (
+  <li className="flex items-start gap-1 my-2">
+    <CheckmarkIcon size={32} className="text-primary shrink-0" />
+    <div className="mt-1">{children}</div>
+  </li>
+);
+
+const Description = styled.div`
+  display: -webkit-box;
+  -webkit-line-clamp: 4;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  flex-shrink: 0;
+  height: 6rem;
+`;
+
+const Price = ({ price }) => (
+  <div className="my-8 text-center">
+    <h2>
+      <sup className="text-lg -top-4">$</sup>
+      {price}
+    </h2>
+    <p className="uppercase text-sm font-light -mt-2">per month</p>
+  </div>
+);
+
+const bandwidth = (value) => parseFloat(bytes(value, { bits: true, binary: true }));
```

Comment:
why do you use parseFloat on this ?

Reply:
So, our limits are _not_ returned in accordance to [SI](https://en.wikipedia.org/wiki/International_System_of_Units#Prefixes) -- we multiply by 1024, not 1000.

Now, `pretty-bytes` allows us to pass a `binary` flag which deals with that, but then it also changes the unit name.
For example, for `downloadBandwidth` we may return `83886080`, which translates either into `83.9 Mbit` (with `binary: false`) or `80 Mibit` (with `binary: true`).

So to get a nice, round value here I'm using the `binary` flag, but also `parseFloat` to get rid of the `Mibit` suffix.

Even though technically `Mibit` is correct, it will confuse users who are used to seeing `Mbps`.

We could fix that by adding a tooltip to the unit. Something like this, but with a nicer tooltip maybe? WDYT?
![image](https://user-images.githubusercontent.com/3853033/159666213-061d78ee-a4bb-4236-ab2d-cafe1292cd90.png)

Reply:
I had a meeting with @ro-tex and @ChrisSchinnerl and we're gonna change the values in accounts to base-10 to reflect the actual limits. You can drop parseFloat for the time being and once the accounts changes are in, we will drop `binary: true` flag too and it should reflect correct value.

Reply:
ok this is already merged in https://github.com/SkynetLabs/skynet-accounts/pull/168

Reply:
c964086f